scripts/thanks.coffee

```diff
@@ -1,3 +1,12 @@
+# Description:
+#   Says thanks
+#
+# Commands:
+#   thanks woz
+#   thanks, woz
+#   thank you woz
+#   thank you, woz
+
 module.exports = (robot) ->
-  robot.hear /[Tt]hanks(,){0,1} [Ww]oz/i, (msg) ->
+  robot.hear /thank(s,?|\s+you,?)\s+woz/i, (msg) ->
```

Comment:
On my cellphone, "thanks" gets auto-corrected to "Thanks".

 It would be nice to still support the capital T version.